service/service.go

```diff
@@ -36,12 +47,16 @@ func DefaultConfig() Config {
 		GitCommit:   "",
 		Name:        "",
 		Source:      "",
+
+		ResourceRetries:           0,
+		ControllerBackOffDuration: time.Duration(0),
```

Comment:
I think we should pass the viper and flag structure until we want to really configure something. Passing all this along through the stack is just not forth it.

Reply:
Yeah, totally. I need to clean this up properly, want to do it in a separate pr